afs/media/js/views/components/reports/textual-work.js

```diff
@@ -37,38 +41,160 @@ define([
+                            statement: 'statement about creation event part',
+                            timespan: 'timespan of creation event part',
                         }
                     },
                 };
+                self.eventCards = {
+                    'publication': {
+                        card: self.cards?.["publication event of textual work"],
+                        subCards: {
+                            name: 'name for publication event',
+                            statement: 'statement of publication event',
+                            timespan: 'timespan of publication event',
+                        }
+                    },
+                };
+                self.substanceData = ko.observable({
+                    sections: [
+                        {
+                            title: "Language", 
+                            data: [{
+                                key: 'language of textual work', 
+                                value: self.getRawNodeValue(self.resource(), 'language'), 
+                                card: self.cards?.['language of textual work'],
+                                type: 'resource'
+                            }]
+                        },
+                        {
+                            title: "Text Content", 
+                            data: [{
+                                key: 'content of textual work', 
+                                value: self.getRawNodeValue(self.resource(), 'content'), 
+                                card: self.cards?.['content of textual work'],
+                                type: 'resource'
+                            }]
+                        }
+                    ]
+                });
+                self.parthoodData = ko.observable({
+                    sections: [
+                        {
+                            title: "Part of Text", 
+                            data: [{
+                                key: 'parent textual work', 
+                                value: self.getRawNodeValue(self.resource(), 'part of'), 
+                                card: self.cards?.['parent textual work'],
+                                type: 'resource'
+                            }]
+                        }
+                    ]
+                });    
+                self.aboutnessData = ko.observable({
+                    sections: [
+                        {
+                            title: "Text Subjects", 
+                            data: [{
+                                key: 'subject(s) of textual work', 
+                                value: self.getRawNodeValue(self.resource(), 'subject'), 
+                                card: self.cards?.["subject(s) of textual work"],
+                                type: 'resource'
+                            }]
+                        },
+                        {
+                            title: "Textual Referent", 
```

Comment:
Think you have a typo here - beyond that, should these be in one "aboutness" section?

Reply:
The titles have been updated with more meaningful names.
I believe these are from two separate cards.